app/styleguides/_default/layouts/layout.css

```diff
@@ -52,10 +52,9 @@ html {
     margin: auto;
   }
 
-  & > .page-header,
   & > .top {
     margin: 20px auto;
-    width: $wrapperMaxWidth;
+    width: 1100px;
```

Comment:
🤔 why the change from variable to specific value?

Reply:
this width was removed in header component PR